shared_model/validators/default_validator.hpp

```diff
@@ -70,6 +71,11 @@ namespace shared_model {
         SignableModelValidator<DefaultBlockValidator,
                                const interface::Block &,
                                FieldValidator>;
+
+    using DefaultBatchValidator =
```

Comment:
It does not validate batch. It actually can be used to validate any collection of transactions which may contain several batches. I would name it smth like `DefaultUnsignedTxCollectionValidator`. I don't know how to make the name shorter